### PR TITLE
feat: preview pluginchart

### PR DESCRIPTION
**Pull request by LiteSun**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bug fix
- [x] New feature provided
- [ ] Improve performance

- Related issues

___
### Bugfix
- Description

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.
added pluginchart preview in route step4

### CODE REVIEW

**Review by juzhiyuan — APPROVED** [review]
LGTM

File: src/pages/Route/components/CreateStep4/CreateStep4.tsx [hunk 1]
```diff
@@ -44,7 +46,12 @@ const CreateStep4: React.FC<Props> = ({ form1, form2, redirect, ...rest }) => {
           <h2 style={style}>{formatMessage({ id: 'route.create.define.api.backend.server' })}</h2>
           <Step2 {...rest} form={form2} disabled />
           <h2 style={style}>{formatMessage({ id: 'route.create.plugin.configuration' })}</h2>
-          <PluginPage data={rest.data.step3Data.plugins} disabled />
+          {Boolean(Object.keys(plugins).length !== 0) && (
+            <PluginPage data={rest.data.step3Data.plugins} disabled />
+          )}
+          {Boolean(Object.keys(script).length !== 0) && (
+            <PluginChart data={rest.data.step3Data.script.chart} readonly onChange={() => {}} />
```
Comment (moonming) [inline src/pages/Route/components/CreateStep4/CreateStep4.tsx:53]: what is `PluginChart`? do you mean `plugin orchestration`?

**Review by moonming — CHANGES_REQUESTED** [review]

File: src/pages/Route/components/CreateStep4/CreateStep4.tsx (hunk 1 above)
Comment (juzhiyuan) [reply src/pages/Route/components/CreateStep4/CreateStep4.tsx:53]: PluginChart is a React component from `api7-dashboard/pluginchart` module, which is used to render chart according to the schema.

Comment (LiteSun) [reply src/pages/Route/components/CreateStep4/CreateStep4.tsx:53]: it seems PluginOrchestration is a better choice, I will change`PluginChart` to `PluginOrchestration` later.

**Review by juzhiyuan — APPROVED** [review]
LGTM

**Comment by juzhiyuan** [comment]
cc @moonming

**Review by moonming — APPROVED** [review]

**Pull request merged**